.storybook/stories/reset.story.jsx

```diff
@@ -8,6 +8,6 @@ import { Reset } from '../../src/reset';
 
 storiesOf('Reset', module).add('default', () => (
   <ThemeWrapper>
-    <Reset />
+    <Reset onAction={() => null} onClose={() => null} />
```

Comment:
@EricTurf rebased on latest `master`, but the only thing I noticed is this new story in the storybook. I chose to keep `isRequired` for the `Reset` component props and use empty functions here in order to catch bugs like the one with `onCancel`.

Reply:
Cool, as far as I know this `reset` story isnt needed. I removed that component completely

Reply:
I'm not sure what you mean, I still see the `src/reset/reset.js` component on `master`. :thinking: